server/data/resources/cheque-requests.json

```diff
@@ -0,0 +1,117 @@
+        "questionMarkdown": "It’s hard to give an exact estimate on how long the process should take but to ensure that the process is as efficient as possible please review the common mistakes in the question below. Feel free to reach out to us at <a href='mailto:vpf@mathsoc.uwaterloo.ca' target='_blank'>vpf@mathsoc.uwaterloo.ca</a>."
+      },
+      {
+        "question": "What are common mistakes that delay the cheque request process?",
+        "questionMarkdown": "<p><ul><li>The preferred name instead of legal name is given on a cheque request form. The person providing the reimbursement needs to provide their name as it would appear with their bank.</li><li>When issuing a cheque request for gift card, the recipients names and emails are not provided. We cannot begin processing the request until we have this information.</li><li>Missing receipts, proof of purchase and event ID number</li><li>Mailing address (as all cheques will be mailed out if they do not get picked up)</li></ul></p>"
+      },
+      {
+        "question": "What happens if my cheque request is denied?",
+        "questionMarkdown": "<p>We will reach out to you if we require more information about your cheque request but as long as your purchase is for an event your club has in their budget and is within budget we should be able to process it. Make sure with gift cards you provide the legal name and email address of the gift card recipients.</p>"
+      },
+      {
+        "question": "What if this is not for a club reimbursement?",
+        "questionMarkdown": "<p>Please see the other MathSoc cheque request form that is used for all non-club events on the website. For any additional questions please email <a href='mailto:vpf@mathsoc.uwaterloo.ca' target='_blank'>vpf@mathsoc.uwaterloo.ca</a>.</p>"
+      },
+      {
+        "question": "How should tips be entered?",
+        "questionMarkdown": "<p>If a tip is given please make a small note under the total tax paid on all purchase line and make sure the club signing officer initials beside the written-in tip. Then include the tip when summing the subtotal and total tax paid on the total line.</p>"
+      },
+      {
+        "question": "What if I have other questions?",
+        "questionMarkdown": "<p>We would love to answer any other questions you may have! Shoot us an email at <a href='mailto:vpf@mathsoc.uwaterloo.ca' target='_blank'>vpf@mathsoc.uwaterloo.ca</a> and we will help you find the answer to your questions.</p>"
+      }
+    ]
+  },
+  "otherForms": {
+    "header": "Other Forms",
+    "footnote": "This form is used to request a refund of the MathSoc fee. It must be filled out in the first month of the term.",
+    "forms": [
+      {
+        "title": "Academic Feedback Form",
+        "link": "https://docs.google.com/forms/d/e/1FAIpQLScl3yNAcyJO0v3z2Pot_PLnVtnsMGHrCFZm4eM90YF6no7SlQ/viewform"
+      },
+      {
+        "title": "Capital Improvements Fund Form",

```

Comment:
Same comment here about adding 'Download'

Reply:
I added it for the cheque request forms, but would "Download" be appropriate here since these are google/office forms that will not be downloaded? Only the pdfs need to be downloaded to be submitted, I think

Reply:
oops yeah you're 100% right, nvm.